app/Models/Relationship/RelationshipType.php

```diff
@@ -79,7 +79,10 @@ public function getLocalizedName(Contact $contact = null, bool $includeOpposite
         }
 
         $femaleVersion = trans('app.relationship_type_'.$this->name.'_female');
-        $maleVersion = trans('app.relationship_type_'.$this->name);
+        $maleVersion = trans('app.relationship_type_'.$this->name.'_male');
+        if ($maleVersion == 'app.relationship_type_'.$this->name.'_male') {
```

Comment:
It's better to use a `===` comparison here.
```suggestion
        if ($maleVersion === 'app.relationship_type_'.$this->name.'_male') {
```